### ISSUE TITLE
3.4.0: NullPointerException in AdViewController.adDidFail()

### ISSUE THREAD

**Issue by jhansche** [mopub/mopub-android-sdk#150]
We're seeing this new crash as of updating to MoPub 3.4.0:
http://crashes.to/s/7a04f254af4
```
java.lang.NullPointerException
       at com.mopub.mobileads.AdViewController.adDidFail(AdViewController.java:440)
       at com.mopub.mobileads.AdViewController.onAdLoadError(AdViewController.java:167)
       at com.mopub.mobileads.AdViewController$1.onErrorResponse(AdViewController.java:115)
       at com.mopub.volley.Request.deliverError(Request.java:570)
       at com.mopub.volley.ExecutorDelivery$ResponseDeliveryRunnable.run(ExecutorDelivery.java:101)
       at android.os.Handler.handleCallback(Handler.java:605)
       at android.os.Handler.dispatchMessage(Handler.java:92)
       at android.os.Looper.loop(Looper.java:137)
       at android.app.ActivityThread.main(ActivityThread.java:4482)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:511)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:794)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:561)
       at dalvik.system.NativeStart.main(NativeStart.java)
```

The NPE occurs at:
```java
    void adDidFail(MoPubErrorCode errorCode) {
        MoPubLog.i("Ad failed to load.");
        setNotLoading();
        scheduleRefreshTimerIfEnabled();
        getMoPubView().adFailed(errorCode); // <---
    }
```

It appears that `mMoPubView` is null, which should only happen after calling `cleanup()`.  Once cleanup is called, nothing else should attempt to interact with the view.

**Comment by joeblubaugh** [comment]
Thanks for the report. We'll release a fix for this issue very soon.

**Comment by joeblubaugh** [comment]
Fixed in 3.5.0
